src/core/createFrameResponse.ts

```diff
@@ -0,0 +1,58 @@
+import { NextResponse } from 'next/server';
+import { Button, FrameMetadata } from './sharedTypes';
+
+/**
```

Comment:
Btw we are missing unit-tests for this utility, can you please add them and cover all different cases.

Reply:
Added test file